### PR TITLE
milestone 3:Contact form

### PR DESCRIPTION
**Pull request by kelwilson**
### contact form ;

- Add background-image to the right top of the page
- Add h2 and a P tag with text 
- Add a form for field
- Add a button for submitting the form
- Add social media handles images
- Add styles to the page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -237,5 +233,61 @@ <h3 class="large">Skills</h3>
...
+            <div class="form-container">
+              <label class="form-label" for="message"></label>
+            <textarea class="text-area" id="message" name="Message" required rows="10" maxlength="500" placeholder="Enter your message here..."></textarea>
+            </div>
+
+        <div class="start-btn-div">
+          <button type="submit" class="start-btn-2">Start collaboration</button>
+        </div>
+
+        </form>
```
Comment (prolajumokeoni) [inline index.html:269]: ### OPTIONAL

- [ ] Kindly center your form so it is always at the center no matter the mobile screensize, everything looks good at 320px, but as I move to larger mobile screen sizes your form looks aligned to the left, see screenshot below 👍🏿 👍🏿 
Screenshot_135

File: index.html [hunk 2]
```diff
@@ -237,5 +233,61 @@ <h3 class="large">Skills</h3>
...
+            <input type="email" class="email" id="email" name="email"  placeholder="Your email" required>
+            </div>
+
+            <div class="form-container">
+              <label class="form-label" for="message"></label>
+            <textarea class="text-area" id="message" name="Message" required rows="10" maxlength="500" placeholder="Enter your message here..."></textarea>
+            </div>
+
+        <div class="start-btn-div">
+          <button type="submit" class="start-btn-2">Start collaboration</button>
```
Comment (prolajumokeoni) [inline index.html:266]: - [x] Kindly add button interactions such as hover, active (when it's pressed), disabled, and enable (default state) as styled  in the Figma template see screenshot below 🤝 .

 
Screenshot_136

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Team
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Kindly ensure that as partners you always have balanced commits in pair programming project 👍 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (kelwilson) [reply index.html:269]: please how would I center the form, don't really understand that, I'm i supposed to use 
 justify-content: center; ? @prolajumokeoni

Comment (kelwilson) [reply index.html:269]: I tried setting the text-align to the center. didn't know of a better way to do it, I hope it works perfect now @prolajumokeoni

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kelwilson** [comment]
please is there any other way I could have positioned my form in the middle without using the text-align property @CollinsTatang

**Comment by CollinsTatang** [comment]
> text-align property

Hi @kelwilson 
I think text-align property is the best for a start but you can read about more options [here](https://www.w3schools.com/cssref/css3_pr_text-align-last.asp)

**Pull request merged**